### PR TITLE
feat: make sure submit interaction also validates inputs

### PR DESCRIPTION
**Pull request by garethloot**
- make sure submit interaction also validates inputs

### CODE REVIEW

**Comment by jeroenreijs** [comment]
We will pick this up with ticket: https://bettyblocks.atlassian.net/browse/TA-2301

**Comment by BarryBlocks** [comment]
@garethloot Please elaborate on the problem, and how this is solved in this PR.

File: src/components/form.js [hunk 1]
```diff
@@ -47,10 +47,7 @@
         const mounted = useRef(false);
 
         B.defineFunction('Submit', () => {
-          if (formRef.current)
-            formRef.current.dispatchEvent(
-              new Event('submit', { cancelable: true }),
-            );
+          if (formRef.current) formRef.current.requestSubmit();
```
Comment (jeroenreijs) [inline src/components/form.js:50]: Apparently `requestSubmit` seems not to be supported in Safari, can you check? https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/requestSubmit.

**Comment by vdberganton** [comment]
Closing due to inactivity

**Pull request closed without merging**